e2e/testinfra/README.md

```diff
@@ -28,22 +25,6 @@ value is `cs-dev-hub`.
 
 ## Usage
 
-
-1. Push test OCI images to private registry, for example,
-    ```bash
-    GCP_PROJECT=your-gcp-project-name make push-test-oci-images-private
```

Comment:
It looks like we might still need this step to push GCR images, right?

Reply:
Good catch. Updated.